container/apbf/example_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2021 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package apbf_test
+
+import (
+	"fmt"
+
+	"github.com/decred/dcrd/container/apbf"
+)
+
+// This example demonstrates creating a new APBF, adding items to it up to the
+// maximum capacity, querying the oldest item, then adding more items to it in
+// order to cause the older items to best evicted.
```

Comment:
s / ... to best evicted / to be evicted.

Reply:
Fixed.